modules/heat_conduction/src/constraints/ModularGapConductanceConstraint.C

```diff
@@ -29,12 +29,14 @@ ModularGapConductanceConstraint::validParams()
                                                "List of GapFluxModel user objects");
   params.addCoupledVar("displacements", "Displacement variables");
 
-  MooseEnum gap_geometry_type("PLATE CYLINDER SPHERE", "PLATE");
-  params.addParam<MooseEnum>("gap_geometry_type",
-                             gap_geometry_type,
-                             "Gap calculation type. The geometry type is used to compute "
-                             "gap distances and scale fluxes to ensure energy balance.");
-  params.addRangeCheckedParam<Real>("max_gap", 1e6, "max_gap>=0", "A maximum gap size");
+  MooseEnum gap_geometry_type("AUTO PLATE CYLINDER SPHERE", "AUTO");
+  params.addParam<MooseEnum>(
+      "gap_geometry_type",
+      gap_geometry_type,
+      "Gap calculation type. The geometry type is used to compute "
```

Comment:
This is a great message that will only be seen by those users that choose to select or investigate this input parameter. At some point we may think of adding some more information output: Would it make sense to make this parameter required? 
via the problem coordinate system -> via the mesh coordinate system

Reply:
I think auto is a good compromise, but happy to do whatever makes sense

Reply:
I am good with your choice here. Just thinking out loud that not all users will get to see your message.